### PR TITLE
Add more parameters to vLLM tutorial

### PR DESCRIPTION
**Pull request by dyastremsky**
- Remove unused class
- Add additional parameters to vLLM tutorial (pulled the integers, floats, and booleans from vLLM's [SamplingParams class](https://github.com/vllm-project/vllm/blob/0967102c6de874902d973f0bcb98d48149a8eb49/vllm/sampling_params.py#L15)

### CODE REVIEW

File: Quick_Deploy/vLLM/model_repository/vllm/1/model.py [hunk 1]
```diff
@@ -127,12 +127,21 @@ def get_sampling_params_dict(self, params_json):
...
         for k in float_keys:
             if k in params_dict:
                 params_dict[k] = float(params_dict[k])
 
+        int_keys = ["best_of", "max_tokens", "n", "top_k"]
+        for k in int_keys:
+            if k in params_dict:
+                params_dict[k] = int(params_dict[k])
+
         return params_dict
```
Comment (rmccorm4) [inline Quick_Deploy/vLLM/model_repository/vllm/1/model.py:145]: Nice addition. Can you double check the `request.parameters()` path works?

From what I can tell we're doing this for the non-input tensor path:
```
parameters = request.paramters() # dictionary
json.loads(parameters) # ERROR: expected string?
```

Comment (dyastremsky) [reply Quick_Deploy/vLLM/model_repository/vllm/1/model.py:145]: Thanks! That path seems to work fine locally. Perhaps parameters is returning a string or json.loads can handle a dictionary input?

Comment (nnshah1) [reply Quick_Deploy/vLLM/model_repository/vllm/1/model.py:145]: I think `request.parameters()` is always a string

Comment (rmccorm4) [reply Quick_Deploy/vLLM/model_repository/vllm/1/model.py:145]: Ah yes, you're right: https://github.com/triton-inference-server/python_backend/tree/main#inference-request-parameters. I always use it via json loads() without thinking.

Thanks for checking!

**Review by rmccorm4 — APPROVED** [review]

**Comment by dyastremsky** [comment]
Thanks for reviewing and approving, Ryan and Neelay! Could someone with tutorials permissions merge this?

CC: @tanayvarshney in case I can get added as a contributor to the tutorial repository.

**Pull request merged**